recipe/meta.yaml

```
@@ -1,17 +1,18 @@
 {% set name = "htcondor" %}
-{% set version = "9_0_7" %}
+{% set version = "9.0.7" %}
 
 package:
   # the top-level package should be called `htcondor`, but
   # because the metapackage needs to specify host dependencies
   # we have to rename the top-level package so that conda-build
   # doesn't get confused
   name: {{ name|lower }}-build
-  version: {{ version|replace('_', '.') }}
+  version: {{ version }}
 
 source:
   url: https://github.com/{{ name }}/{{ name }}/archive/V{{ version }}.tar.gz
-  sha256: 561a600d8c7848a85c3f46da949300c2f1e90c1e028e574b1760158086f54f35
+  url: https://research.cs.wisc.edu/htcondor/tarball/stable/{{ version }}/release/condor-{{ version }}-src.tar.gz
```

```suggestion
  url: https://research.cs.wisc.edu/htcondor/tarball/9.0/{{ version }}/release/condor-{{ version }}-src.tar.gz
```

```suggestion
  url: https://research.cs.wisc.edu/htcondor/tarball/{{ version.split('.')[:2] | join('.') }}/{{ version }}/release/condor-{{ version }}-src.tar.gz
```

There is no more `stable`. Either put the LTS version here directly and update as needed, or use the template (with the warning that things are likely to break between major LTS version changes).